tools/devtool

```diff
@@ -822,11 +822,21 @@ mount_ramdisk() {
     umount_ramdisk
     mkdir -p ${DEFAULT_RAMDISK_PATH} && \
     mount -t tmpfs -o size=${ramdisk_size} tmpfs ${DEFAULT_RAMDISK_PATH}
+    ok_or_die "Failed to mount ramdisk to ${DEFAULT_RAMDISK_PATH}. Check the permission."
     mkdir -p ${DEFAULT_RAMDISK_PATH}/srv
     mkdir -p  ${DEFAULT_RAMDISK_PATH}/tmp
 }
 
 umount_ramdisk() {
+    if [ ! -e "${DEFAULT_RAMDISK_PATH}" ]; then
```

Comment:
this check is not needed if we later check that `[ -d "${DEFAULT_RAMDISK_PATH}" ]`

Reply:
There three patterns of `DEFAULT_RAMDISK_PATH`.
- not exist
- file
- directory

Obviously, if the path does not exist, we must not do nothing, just return 0.
If the path is a file, `DEFAULT_RAMDISK_PATH` might be wrong, so it had better die.
If the directory is a directory, it should unmount directory and remove it.

This if clause is for checking the existence of the path.
The second if clause is for checking whether the path is a file or a directory.

Reply:
thanks for the explanation